components/bh1750/include/bh1750.h

```diff
@@ -4,6 +4,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/**
+ * @file
+ * @brief BH1750 driver
+ */
```

Comment:
Note that file-level comments are only used when Doxygen generates HTML output; when we use Doxygen XML files along with Breathe or another tool, this will probably not be used.

Reply:
Sure, we can keep that.